pkg/config/config.go

```diff
@@ -36,13 +35,17 @@ func (c Config) String() string {
 }
 
 // Load parses the YAML input s into a Config.
-func Load(s string) (*Config, error) {
+func Load(b []byte) (*Config, error) {
+	if len(b) == 0 {
+		return nil, nil
+	}
+
+	// TODO: Sanitize the input labels if it becomes a problem for the user.
+	// https://github.com/prometheus/prometheus/blob/1bfb3ed062e99bd3c74e05d9ff9a7fa4e30bbe21/util/strutil/strconv.go#L51
 	cfg := &Config{}
 
-	dec := yaml.NewDecoder(bytes.NewBufferString(s))
-	dec.KnownFields(true)
```

Comment:
Why was this removed?

Reply:
We are using a different API and it doesn't have this method.